clients/go/admin/deviceflow/config.go

```diff
@@ -7,4 +7,5 @@ type Config struct {
 	TokenRefreshGracePeriod config.Duration `json:"refreshTime" pflag:",grace period from the token expiry after which it would refresh the token."`
 	Timeout                 config.Duration `json:"timeout" pflag:",amount of time the device flow should complete or else it will be cancelled."`
 	PollInterval            config.Duration `json:"pollInterval" pflag:",amount of time the device flow would poll the token endpoint if auth server doesn't return a polling interval. Okta and google IDP do return an interval'"`
+	Audience                string          `json:"audience" pflag:",audience value to be passed when requesting access token.Required when auth server hasn't been configured with default audience"`
```

Comment:
Should this come from the GetPublicClient server API?

Reply:
Makes sense . This makes admin control it better instead of allowing  any  value being passed from flytectl

Reply:
Done